handler.go

```diff
@@ -69,6 +70,7 @@ type respError struct {
 	Code    ErrorCode       `json:"code"`
 	Message string          `json:"message"`
 	Meta    json.RawMessage `json:"meta,omitempty"`
+	Data    interface{}     `json:"data,omitempty"`
```

Comment:
Why is this `interface{}` ?

Reply:
@aarshkshah1992 I posted a comment above in current PR regarding this: https://github.com/filecoin-project/go-jsonrpc/pull/123#issuecomment-2402626162

Reply:
Can we make this just a string ? Why does it have to be `json.RawMessage` or `interface{}`

@rvagg  -> Any thoughts ?

Reply:
https://www.jsonrpc.org/specification#error_object

> A Primitive or Structured value that contains additional information about the error.

:shrug: `interface{}` would seem more accurate, as long as it's serialisable